--- /dev/null
+++ ../Helper/Helper.php
@@ -47,12 +47,12 @@
     {
         $string ??= '';
 
-        if (preg_match('//u', $string)) {
-            $string = preg_replace('/[\p{Cc}\x7F]++/u', '', $string, -1, $count);
+//        if (preg_match('//u', $string)) {
+//            $string = preg_replace('/[\p{Cc}\x7F]++/u', '', $string, -1, $count);
+//
+//            return (new UnicodeString($string))->width(false) + $count;
+//        }
 
-            return (new UnicodeString($string))->width(false) + $count;
-        }
-
         if (false === $encoding = mb_detect_encoding($string, null, true)) {
             return \strlen($string);
         }
@@ -68,9 +68,9 @@
     {
         $string ??= '';
 
-        if (preg_match('//u', $string)) {
-            return (new UnicodeString($string))->length();
-        }
+//        if (preg_match('//u', $string)) {
+//            return (new UnicodeString($string))->length();
+//        }
 
         if (false === $encoding = mb_detect_encoding($string, null, true)) {
             return \strlen($string);
@@ -86,9 +86,9 @@
     {
         $string ??= '';
 
-        if (preg_match('//u', $string)) {
-            return (new UnicodeString($string))->slice($from, $length);
-        }
+//        if (preg_match('//u', $string)) {
+//            return (new UnicodeString($string))->slice($from, $length);
+//        }
 
         if (false === $encoding = mb_detect_encoding($string, null, true)) {
             return substr($string, $from, $length);
